content/02.introduction.md

```diff
@@ -155,14 +155,23 @@ Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly di
+
+People living in certain locations may be especially vulnerable to harm. In a preprint, [@doi:10.1101/2020.04.05.20054502] provided observational evidence that geographical areas in the United States which suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates. Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).  Individuals in nursing homes / skilled nursing facilities [@doi:10.1001/jama.2020.11642] and in some prisons and detention centers [@doi:10.1001/jamainternmed.2020.1856; @doi:10.1001/jama.2020.12528] have also been exposed to higher risk of infection.
+Certain occupations, such as health care work, can put individuals at increased risk [@doi:10.1002/ajim.23145; 
+@doi:10.1371/journal.pone.0232452; @url:www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020]
+
+Many other factors influence risk of serious COVID-19 outcomes.  Genetic factors may play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101].  Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].  [@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.  Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042]. Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].
+
+Study of disparities and differences among groups in rates of hospitalization or death from COVID-19 are complicated by the fact that risk may be elevated in multiple ways.  An individual may be more likely to contact the virus, more likely to get infected once exposed, be more likely to have serious complications once infected, or be less likely to get adequate help once they are seriously ill. Although these are difficult to disaggregate, they suggest different possible interventions and each deserves consideration.
```

Comment:
typo: "An individual may be more likely to": "contract" instead of "contact"

Reply:
Thanks!  I think that by "contact" I really meant "contact" (be exposed to), but this turns out to be a terrible choice of words because it looks like a typo for "contract."  So I changed "contact" to "be exposed to."